### PR TITLE
disabled inputs

### PR DESCRIPTION
**Pull request by jonpitch**

### CODE REVIEW

**Comment by jonpitch** [comment]
This PR includes #8

File: addon/components/impact-question-select.js [hunk 1]
```diff
@@ -12,25 +12,25 @@ export default Component.extend({
...
   // default to service if no theme is supplied
   themeName: computed('useTheme', function() {
     const theme = this.get('useTheme');
     return isEmpty(theme) ? this.get('theme.name') : theme;
   }),
 
   actions: {
+
+    // handle selection
     onSelectionChanged(selectedValue) {
```
Comment (kikemoto) [inline addon/components/impact-question-select.js]: @jonpitch What do you think about returning if `disabled` is true? Similar to what's in `components/impact-question-radio` line 29.

**Comment by jonpitch** [comment]
@kikemoto good question about targets/bookmarks. I wasn't sure what should happen since the consumer is providing those actions. Any ideas?

**Comment by kikemoto** [comment]
@jonpitch What if the `tooltipped` icon supports a `disabled` property and when `disabled` it does not perform its action? The `question-card` would set `disabled` on the yielded target and bookmark components based on `editable`. If for some reason callers always want their action performed they could override `disabled` when they write the yielded component.

**Comment by jonpitch** [comment]
@kikemoto good thought as well. I initially left it out assuming callers might want their action always called; if they can override than that's perfect.

**Comment by jonpitch** [comment]
@kikemoto `ember-chosen` I worked around but it'll do for now and the `tooltipped-icon` can be disabled, which just prevents firing the action. We'll probably want to propagate `editable` from the `question-card` down to verification as well; but let's discuss when that comes into focus.

**Pull request merged**